kong/templates/nginx_kong.lua

```diff
@@ -453,7 +453,11 @@ server {
 > if not legacy_worker_events then
 server {
     server_name kong_worker_events;
+> if resty_events_sock_path and #resty_events_sock_path > 0 then
+    listen unix:${{RESTY_EVENTS_SOCK_PATH}}/worker_events.sock;
+> else
     listen unix:${{PREFIX}}/worker_events.sock;
+> end
```

Comment:
if we construct the value in the config loader, we can just set it here, no need for the if/then logic in the templates (this one as well as the stream one)

Reply:
good idea, but I am not familiar with conf loader's code.

Reply:
I add some code in conf loader, now we use a simple `${{RESTY_EVENTS_SOCK_PATH}}`.